test/runtest.py

```diff
@@ -2842,3 +2842,18 @@ def f():
         return 1
 
     assert ray.get(f.remote()) == 1
+
+
+def test_runtime_context(shutdown_only):
```

Comment:
we can change `runtest.py/test_specific_driver_id` to use runtime context, then we don't need to add this test.